### DISCUSSION TITLE
Dynamic Highlights or this plugin

### DISCUSSION THREAD

**Discussion by zanodor** [Mara-Li/obsidian-regex-mark#18] [General]
Which plugin is better for usability and performance? Any lags?

**Comment by Mara-Li** [comment]
I didn't notice anything on my side! I use it a lot because I have the "discord reflex" to set underline with `__` with I set on my own configuration.
I also set spoiler, but I don't use it a lot.

My own configuration looks like this:
![image](https://github.com/user-attachments/assets/0bf01908-4516-4f2a-b087-3189b32643ab)
Some doesn't are efficient as regex because it spots during normal obsidian markdown (hello footnote), but it's more regex than the plugin.

As far I know, the plugin will be problematic with latex (see #15) and maybe have some bug with table. 

I tried to prevent the maximum crash of cm/obsidian and rendering, but it could break during update of Obsidian.

If you see some lag/performance problem, don't hesitate to make a bug report!
